### ISSUE TITLE
Member accessibility

### ISSUE THREAD

**Issue by parloti** [parloti/FluentRegex#1]
https://github.com/parloti/TypeRegex/blob/ef7fb8f62d3854b39eaab07401fe2131241b696d/TypeRegex/Pattern.cs#L50

Member accessibility should be private.
